src/quartz/scene/doodad/Doodad.cpp

```diff
@@ -91,22 +101,32 @@ quartz::scene::Doodad::~Doodad() {
 }
 
 void
-quartz::scene::Doodad::fixedUpdate() {
+quartz::scene::Doodad::fixedUpdate(
+    const quartz::managers::InputManager& inputManager
+) {
     /**
-     * @todo 2024/06/20 Call the fixed update callback given to the doodad which actually contains
-     *    the logic that we execute here
-     *
      * @todo 2024/06/21 Update m_currentTransform here????
      */
 
+    if (m_fixedUpdateCallback) {
+        auto safeFixedUpdateCallback = [&] () { m_fixedUpdateCallback(this, inputManager); };
+        safeFixedUpdateCallback();
+    }
```

Comment:
Adhere to the comments about setting a no-op callback if the provided ones are not good